tests/pytests/test_config.py

```diff
@@ -164,6 +167,7 @@ def test_arg_num(arg_name, arg_value):
     test_arg_num('UNION_ITERATOR_HEAP', 20)
     test_arg_num('_NUMERIC_RANGES_PARENTS', 1)
     test_arg_num('BG_INDEX_SLEEP_GAP', 15)
+    test_arg_num('MINSTEMLEN', 3)
```

Comment:
Didn't you set its default to be `4` 🤔 ?

Reply:
Yes. But `test_arg_num` is not using the default value, it is passing  `MINSTEMLEN 3` as module argument